paradise_papers_django/manage.py

```diff
@@ -3,7 +3,7 @@
 import sys
 
 if __name__ == "__main__":
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "paradise_papers_django.settings")
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "paradise_papers_django.settings.local")
```

Comment:
could you let me know what does that local does? Thanks :)

Reply:
Hey I should put `dev` instead of `local` 😅 
This will be the default setting the `local` file contain all settings that we need on our local environment. I put it as a default because this will be transparent for the developers to change it we only need to update the  `setting` variable like this
`python manage.py runserver --settings=paradise_papers_django.settings.production`